robot_skills/src/robot_skills/hero.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, wait_services=False):
 
         self.add_arm_part(
             'leftArm',
-            arms.ForceSensingArm(self.robot_name, self.tf_listener, self.get_joint_states, "left")
+            arms.SuctionGripperArm(self.robot_name, self.tf_listener, self.get_joint_states, "left")
```

Comment:
Currently an arm can only be a forcesensingArm or a SuctionGripperArm. This is a problem but not the problem for this PR. Until then either solution misses out on functionality. Luckily neither are really used in the common smach states

Reply:
Reverted this change for now. But a change is needed to make the robot arm part composable of different functionality